app/views/multiyear/how_it_works.scala.html

```diff
@@ -35,8 +35,9 @@ <h1 class="heading-xlarge">@Html(Messages("generic.ma.apply"))</h1>
     </header>
 
     <p>
-    @Html(Messages("pages.how-it-works.lede-pre1",
-    NumberFormat.getIntegerInstance().format(MAX_ALLOWED_PERSONAL_ALLOWANCE_TRANSFER())))</p>
+        <a href=@marriageAllowanceGuideUrl>@messages("pages.how-it-works.lede-pre1.link-text")</a>
+        @messages("pages.how-it-works.lede-pre1",
```

Comment:
Needs HTML tag as the welsh message includes an ascii code. Alternatively, replace the ascii code with its character equivalent. Same for all instances of this message key

Reply:
replaced ascii code